### ISSUE TITLE
Incorrect error message hint for InvalidValueError of std::duration

### ISSUE THREAD

**Issue by tdolsen** [edgedb/edgedb#3641]
```
edgedb> select <duration>"1 day";
edgedb error: InvalidValueError: invalid input syntax for type std::duration: '1 day'
  Hint: Units bigger than days cannot be used for std::duration.
```

**Expected**: Hint should say "hours", not "days".

**Comment by aljazerzen** [comment]
I've updated the hint:
```
edgedb> select <duration>"1 days";
edgedb error: InvalidValueError: invalid input syntax for type std::duration: '1 days'
  Hint: Day, month and year units cannot be used for std::duration.
```

**Comment by msullivan** [comment]
Oh, heh, the bug got fixed in aed6f984814992f04328cee0496899c2e5bb4382 but we never closed this
